contracts/svm/programs/express_relay/src/sdk/test_helpers.rs

```diff
@@ -0,0 +1,129 @@
+use {
+    crate::{
+        accounts,
+        instruction,
+        InitializeArgs,
+        SubmitBidArgs,
+        ID as EXPRESS_RELAY_PID,
+        SEED_CONFIG_ROUTER,
+        SEED_METADATA,
+    },
+    anchor_lang::{
+        prelude::*,
+        solana_program::{
+            instruction::Instruction,
+            sysvar::instructions as sysvar_instructions,
+        },
+        system_program,
+        InstructionData,
+    },
+};
+
+// This file contains setup helpers for integrating programs to set up express relay for Rust-based tests
+
+// Helper method to create an instruction to initialize the express relay program
+// Should be able to sign transactions with the secret keys of the provided payer and relayer_signer
+// The fee split is set to 100% for the router, since fee payments to relayer are not important for the integrating protocol's tests
+pub fn create_initialize_express_relay_ix<'info>(
+    payer: Pubkey,
+    admin: Pubkey,
+    relayer_signer: Pubkey,
+    fee_receiver_relayer: Pubkey,
+) -> Instruction {
+    let express_relay_metadata =
+        Pubkey::find_program_address(&[SEED_METADATA], &EXPRESS_RELAY_PID).0;
+
+    let split_router_default = 10000;
+    let split_relayer = 0;
+
+    let accounts_initialize = accounts::Initialize {
+        payer,
+        express_relay_metadata,
+        admin,
+        relayer_signer,
+        fee_receiver_relayer,
+        system_program: system_program::ID,
+    }
+    .to_account_metas(None);
+    let data_initialize = instruction::Initialize {
+        data: InitializeArgs {
+            split_router_default,
+            split_relayer,
+        },
+    }
+    .data();
+
+    Instruction {
+        program_id: EXPRESS_RELAY_PID,
+        accounts:   accounts_initialize,
+        data:       data_initialize,
+    }
+}
+
+// Creates and adds a SubmitBid instruction to the provided instructions
+pub fn add_express_relay_submit_bid_instruction(
+    ixs: &mut Vec<Instruction>,
```

Comment:
isn't it better to get a transaction?

Reply:
making the transaction involves setting payers and signers, and i didn't want to expose that here. you could create an unsigned transaction with `new_with_payer`, but i think it would be easier to deal with a vec/array of instructions. the integrating program can transform that into a Transaction however they see best